src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -118,6 +118,7 @@ const messageRight = css`
 export const ShoutOutMessage = styled.div<ReactionProps>`
   bottom: 0;
   position: absolute;
+  z-index: 1;
```

Comment:
Is this needed here? Based on the natural stack ordering I believe it shouldn’t be, as the shoutout is rendered after the reactions?

I don’t think I ended up with any Z indexes in my WIP refactor PR in the end (though perhaps there is an edgecase I missed that’s still broken there)

Reply:
Without this, other peoples' shoutouts are behind the emojis(probably because they are in different relative containers).

Reply:
@0xdevalias shall I merge this into the prod to unblock our sales?
It's probably a hacky way, but it works. Given that the PR to have it clean will be merged in the nearest days

Reply:
One the edge cases is to have the user(1) shoutout over user(2)emojis, since the natural stack layering doesn't seem to work in this case

Reply:
Makes sense, I’ll have to check mine closer.

Yeah, if this is tested and resolves things, definitely get it shipped out across all the prod envs.